chrome-2023-experiment-overview.md

```diff
@@ -0,0 +1,101 @@
+
+## Opting into the origin trial
+
+Developers can sign up for an [origin trial](https://developer.chrome.com/docs/web-platform/origin-trials/) token.
+
+> **Note**
+> This trial is not yet available. We'll update this with a link when it launches.
+
+Most developers will only need a token for their own origin, and can use it the usual way.
+
+```http
+Origin-Trial: [token issued to your origin]
+```
+
+```html
+<meta http-equiv="origin-trial" content="[token issued to your origin]">
+```
+
+For this trial, Chrome will also accept a third-party token issued to another origin, as long as either:
+
+* an external script loaded from that origin adds it, or
+* the server delivers the document response with a `Speculation-Rules` HTTP response header which loads speculation rules and an `Origin-Trial` HTTP response header issued to that origin
+
+```http
+Origin-Trial: [third-party token for https://rules-provider.example]
+Speculation-Rules: "https://rules-provider.example/speculationrules.json"
+```
+
+## HTTP-fetched speculation rules
+
+Some developers prefer to deliver speculation rules out of line. You can do this using the `Speculation-Rules` response header.
+
+```http
+Speculation-Rules: "/speculationrules.json"
+```
+
+This will be fetched and cached like any other subresource, though at a lower priority since it doesn't block the current page load. This resource must use the correct MIME type and, if it is cross-origin, pass a [CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS) check.
+
+```http
+Content-Type: application/speculationrules+json
+Access-Control-Allow-Origin: *
+```
+
+## Automatic link finding
+
+With this enhancement, the browser can automatically react find links in your page which the user is likely to click. The developer simply needs to communicate which links are appropriate to prefetch, since some servers might exhibit side effects for `GET` requests.
+
+You can do this by using [URL patterns](https://developer.mozilla.org/en-US/docs/Web/API/URL_Pattern_API) and simple boolean expressions using "and", "or", and "not". Support for selecting links using CSS selectors is planned. (Let us know if this would make a difference to you!)
+
+```json
+{"prefetch": [
+   {"source": "document",
+    "where": {"or": [
+       {"href_matches": "/articles/*\\?*",
+        "relative_to": "document",
+        "referrer_policy": "strict-origin-when-cross-origin"},
+       {"href_matches": "/products/*\\?*",
+        "relative_to": "document",
+        "referrer_policy": "strict-origin-when-cross-origin"}
+    ]}}
+]}
+```
+
+## When automatic prefetching occurs
+
+In Chrome 110, this experiment will cause Chrome to prefetch up to some small number (currently 5) of URLs when the pointer is down (i.e., mouse down or touch start), which typically occurs between 100 and 200 milliseconds before click, if your speculation rules permit it.
```

Comment:
I find this sentence a bit convoluted. Here's my attempt:

> In Chrome 110, when the pointer is down on an `<a>` element that points to a URL that appears in a `"document"`-source speculation rule, Chrome will start prefetching that link. "Pointer is down" means either mouse down, or touch start, and typically this occurs between 100 and 200 milliseconds before a click.

(we could mention `<area>` too, assuming that's supported? But probably not worthwhile.)